pkg/kosmosctl/install/install.go

```diff
@@ -185,118 +209,146 @@ func (o *CommandInstallOptions) runClusterlink() error {
 				return fmt.Errorf("kosmosctl install clusterlink run error, crd options failed: %v", err)
 			}
 		}
+		klog.Info("Create CRD " + crds.Items[i].Name + " successful.")
 	}
 
-	deployment, err := util.GenerateDeployment(manifest.ClusterlinkNetworkManagerDeployment, manifest.DeploymentReplace{
+	klog.Info("Start creating kosmos-clusterlink Deployment...")
+	clusterlinkDeployment, err := util.GenerateDeployment(manifest.ClusterlinkNetworkManagerDeployment, manifest.DeploymentReplace{
 		Namespace:       o.Namespace,
 		ImageRepository: o.ImageRegistry,
 		Version:         version.GetReleaseVersion().PatchRelease(),
 	})
 	if err != nil {
 		return err
 	}
-	_, err = o.Client.AppsV1().Deployments(o.Namespace).Create(context.Background(), deployment, metav1.CreateOptions{})
+	_, err = o.Client.AppsV1().Deployments(o.Namespace).Create(context.Background(), clusterlinkDeployment, metav1.CreateOptions{})
 	if err != nil {
 		if !apierrors.IsAlreadyExists(err) {
 			return fmt.Errorf("kosmosctl install clusterlink run error, deployment options failed: %v", err)
 		}
 	}
+	if err = util.WaitDeploymentReady(o.Client, clusterlinkDeployment, 120); err != nil {
+		return fmt.Errorf("kosmosctl install clusterlink run error, deployment options failed: %v", err)
+	} else {
+		klog.Info("Deployment clusterlink-network-manager has been created.")
+	}
 
 	return nil
 }
 
-func (o *CommandInstallOptions) runClusterrouter() error {
+func (o *CommandInstallOptions) runClustertree() error {
+	klog.Info("Start creating kosmos-clustertree...")
 	namespace := &corev1.Namespace{}
 	namespace.Name = o.Namespace
-	_, err := o.Client.CoreV1().Namespaces().Create(context.TODO(), namespace, metav1.CreateOptions{})
-	if err != nil {
-		if !apierrors.IsAlreadyExists(err) {
-			return fmt.Errorf("kosmosctl install clusterrouter run error, namespace options failed: %v", err)
+	_, err := o.Client.CoreV1().Namespaces().Get(context.TODO(), util.DefaultNamespace, metav1.GetOptions{})
+	if err != nil && apierrors.IsNotFound(err) {
```

Comment:
if err is not nill and IsNotFound(err),  there is not code for this situation.

Reply:
I've modified the logic here, is it possible now?